src/Host.h

```diff
@@ -744,7 +748,7 @@ private slots:
     int mHostID;
     QString mHostName;
     QString mDiscordGameName; // Discord self-reported game name
-
+    
```

Comment:
Qt Creator really bugs me when it does this when you paste in/edit something and it automatically inserts indentation on surrounding lines...
```suggestion

```